StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentMethodCollectionView.swift

```diff
@@ -91,11 +103,29 @@ extension SavedPaymentMethodCollectionView {
             button.accessibilityLabel = String.Localized.edit
             return button
         }()
+        lazy var defaultBadge: UILabel = {
+            let label = UILabel()
+            label.font = appearance.scaledFont(for: appearance.font.base.medium, style: .footnote, maximumPointSize: 20)
+            label.textColor = appearance.colors.textSecondary
+            label.adjustsFontForContentSizeCategory = true
+            label.text = STPLocalizedString("Default", "Label for identifying the default payment method.")
+            return label
+        }()
 
         fileprivate var viewModel: SavedPaymentOptionsViewController.Selection?
+        var isDefaultPM: Bool = false
 
         var isRemovingPaymentMethods: Bool = false {
             didSet {
+                if allowsSetAsDefaultPM {
+                    if isRemovingPaymentMethods {
+                        addDefaultBadge()
+                        defaultBadge.isHidden = !isDefaultPM
```

Comment:
I find it a little odd that we add the default badge to PMs that aren't the default then remove them. We should move this logic into the `addDefaultBadge()` function and maybe rename it to `addDefaultBadgeIfNecessary`.